packages/widgets/src/blocks/legacy-widget/transforms.js

```diff
@@ -186,7 +186,27 @@ const legacyWidgetTransforms = [
 } );
 
 const transforms = {
-	to: legacyWidgetTransforms,
+	to: [
+		...legacyWidgetTransforms,
+		{
```

Comment:
Why is this added here like this? Can't it just be part of the legacyWidgetTransforms object?